src/science/mathematics/linear_algebra/deserialize-matrix.md

```diff
@@ -0,0 +1,29 @@
+## (De)-Serialize a Matrix
+[![ndarray-badge]][ndarray] [![cat-science-badge]][cat-science]
+
+Serialize and deserialize a matrix.
```

Comment:
Can you spell out in the description that you are serializing to JSON and include a link to `serde_json::to_string`

Reply:
Certainly :smile: